pkgs/applications/video/subtitles-rs/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, fetchFromGitHub, rustPlatform, makeWrapper }:
+
+rustPlatform.buildRustPackage rec {
+  name = "subtitles-rs";
+  version = "20180611";
```

Comment:
There are releases in the Github repo. Would it be possible to use the latest one?

Reply:
Tried and it failed. See my last message